### PR TITLE
fix(build): migrate to using main branch

### PR DESCRIPTION
**Pull request by sofisl**

### CODE REVIEW

File: endpoints/getting-started-grpc/api_config.jwt.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #
 
 # The configuration schema is defined by service.proto file
-# https://github.com/googleapis/googleapis/blob/master/google/api/service.proto
+# https://github.com/googleapis/googleapis/blob/main/google/api/service.proto
```
Comment (ace-n) [inline endpoints/getting-started-grpc/api_config.jwt.yaml:22]: **N.B:** this is talking about a repo other than `nodejs-docs-samples`, and currently `404`s.

Maybe we should do one of the following? Feel free to use whichever one you'd prefer.
a) manually check that all these links work
b) not change anything outside of `nodejs-docs-samples` for now, and file a follow-up issue

**Review by ace-n — CHANGES_REQUESTED** [review]

**Comment by JustinBeckwith** [comment]
Looks like the failing docs test is very real here :)

File: endpoints/getting-started-grpc/api_config.jwt.yaml (hunk 1 above)
Comment (sofisl) [reply endpoints/getting-started-grpc/api_config.jwt.yaml:22]: Thanks Ace! I fixed the offending links :)

Comment (sofisl) [reply endpoints/getting-started-grpc/api_config.jwt.yaml]: Thanks Ace! I fixed the offending links :)

**Review by ace-n — APPROVED** [review]
LGTM

**Comment by bcoe** [comment]
@sofisl you might need to push an empty commit to kick off tests, I'm betting you got bit by actions being down earlier this week.

**Review by JustinBeckwith — APPROVED** [review]

**Pull request merged**